week_12/case_study_12.Rmd

```diff
@@ -1,7 +1,18 @@
 ---
-title: "Case Study 12"
-author: Your Name
-date: August 1, 2020
+title: "Resource Presentation"
+author: Weishan Bai
+date: Nov 20, 2020
 output: github_document
 ---
- 
\ No newline at end of file
+
```

Comment:
This does not address the case study at all.  Perhaps you haven't updated it yet...